makefile

```
@@ -0,0 +1,15 @@
+.PHONY: test_with_coverage
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)